lib/ansible/module_utils/csharp/Ansible.Basic.cs

```diff
@@ -375,8 +379,18 @@ public void LogEvent(string message, EventLogEntryType logEntryType = EventLogEn
                 }
             }
             if (sanitise)
+            {
                 message = (string)RemoveNoLogValues(message, noLogValues);
-            message = String.Format("{0} - {1}", ModuleName, message);
+            }
+
+            if (String.IsNullOrWhiteSpace(TargetLogInfo))
+            {
+                message = String.Format("{0} - {1}", ModuleName, message);
+            }
+            else
+            {
+                message = String.Format("{0} {1} - {2}", ModuleName, TargetLogInfo, message);
```

Comment:
Missed this last time, but on the Python version, we're masking *after*  the TARGET_LOG_INFO stuff has been spliced in, so for parity, the RemoveNoLogValues check/call should move below this.